.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,32 @@
+---
+name: Bug report
+about: Create a bug report to help us fix things
```

Comment:
Would it make sense to mention that this is our preferred way of doing things. I don't think we want to force this on people just for the sake of it.
i.e. the tone makes it feel like all this is forced. and the only way.

An example where this doesn't apply or is just silly:
**Describe the bug**
My watch bootloops

**To reproduce**
Install AsteroidOS.
Bootloop.

**Expected behavior**
No bootloops

I would say that in this case it doesn't really help. You could say that the above isn't concise. Which is true. But this is something we might get when people don't really want to use this structure. In my opinion it should be a guideline.

So relax the title to something like: In order to help us fix issues we would like you to provide a detailed bug report. Please try to describe your issues as well as possible.